storage/Devices/MdImpl.cc

```diff
@@ -401,6 +414,7 @@ namespace storage
 		else
 		{
 		    Md* md = Md::create(prober.get_system(), name);
+		    md->get_impl().set_active(!entry.inactive);
```

Comment:
Is that correct? The ```active``` flag of BldDevice::Impl "means that the device node exists". That is the case even for RAIDs where the MD subsystem reports them RAID as inactive.